examples/jmespath_functions/src/powertools_json_jmespath_function.py

```diff
@@ -0,0 +1,7 @@
+import powertools_json_jmespath_schema as schemas
+
+from aws_lambda_powertools.utilities.validation import validate
+
+sample_event = {"data": '{"payload": {"message": "hello hello", "username": "blah blah"}}'}
+
+validate(event=sample_event, schema=schemas.INPUT, envelope="powertools_json(data)")
```

Comment:
I think we can do better as part of this refactoring - could you add a Lambda Handler and think of try/exception too (in case validation fails)?

I think we can have `sample_event` as a separate JSON too so we standardize any input as a separate JSON file

Similar comment on Parser as there's a great portion of customers who aren't full time devs

https://github.com/awslabs/aws-lambda-powertools-python/issues/1371#issuecomment-1203080410

Reply:
Hi @heitorlessa. I refactored the code including your suggestions and now we have a real example to help people use the validation method.

I created an example simulating the creation of an Order and I used types like integer, number, string and now we have a complete example to demonstrate the power of this feature.